.github/workflows/install_dependencies_run.sh

```diff
@@ -1,2 +1,2 @@
 apt-get update && apt-get install --no-install-recommends -y libdatetime-perl iverilog python3-pip git \
-    libc6 libffi6 libgcc1 libreadline7 libstdc++6 libtcl8.6 python3 zlib1g libbz2-1.0 rsync
+    libc6 libffi6 libgcc1 libreadline7 libstdc++6 libtcl8.6 python3 zlib1g libbz2-1.0 rsync make python3.8
```

Comment:
Not sure if this is needed or not.

Reply:
This is runtime environment, python3.8 is used while running task scripts

Reply:
But I think our python script are compatible to v3.5+? Is there any restriction introduced?

Reply:
Yes I started using f-strings, which is a feature in python3.8+

Reply:
I am concerned by the back-compatibility. Maybe we should apply python compatibility tests in CI